opensrp-reveal/src/test/java/org/smartregister/reveal/TestListTaskView.java

```diff
@@ -0,0 +1,139 @@
+package org.smartregister.reveal;
+
+import android.content.Context;
+import android.location.Location;
+import android.support.annotation.NonNull;
+import android.support.v4.util.Pair;
+
+import com.mapbox.geojson.Feature;
+import com.mapbox.geojson.FeatureCollection;
+import com.mapbox.geojson.Geometry;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+
+import org.json.JSONObject;
+import org.smartregister.reveal.contract.ListTaskContract;
+import org.smartregister.reveal.model.CardDetails;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Vincent Karuri
+ */
+public class TestListTaskView implements ListTaskContract.ListTaskView {
```

Comment:
Delete this class and use Mockito to mock this class e.g  
`ListTaskContract.ListTaskView listTaskView = mock(ListTaskContract.ListTaskView.class);`

Reply:
Done.